homework_3/homework3.py

```diff
@@ -5,11 +5,12 @@
 age_in_month = 12*17
 print("month =", age_in_month) 
 #3
-age_in_years = 204/12
-print("years",age_in_years)
+age_in_years = int(age_in_month/12)
+print("years =",age_in_years)
 #4
-my_age = "Му name is Dmytro and I’m 17 years old"
-print(my_age)
+my_age = "Му name is Dmytro and I’m" ,age_in_years," years old"
```

Comment:
To concatenate strings, it is better to use `+` sign. Replace commas here with `+`